xlive/H2MOD.cpp

```diff
@@ -881,7 +877,7 @@ void H2MOD::CustomSoundPlay(const wchar_t* soundName, int delay)
 		if (delay > 0)
 			std::this_thread::sleep_until(timePoint);
 
-		PlaySound(soundName, NULL, SND_FILENAME);
+		PlaySound(soundName, NULL, SND_FILENAME | SND_NODEFAULT);
```

Comment:
when is soundName null/empty? why even go through any of this code if a null soundName was passed in? check add a null check and return